### PR TITLE
Kill some trailing whitespace, update to C# 6

### PR DESCRIPTION
**Pull request by khellang**
What the title says. Closes (most of?) #601. Was cruising by anyway.

### CODE REVIEW

File: src/Serilog/Context/LogContext.cs [hunk 1]
```diff
@@ -94,8 +95,7 @@ public static IDisposable PushProperties(params ILogEventEnricher[] properties)
             var stack = GetOrCreateEnricherStack();
             var bookmark = new ContextStackBookmark(stack);
 
-            foreach (var prop in properties)
-                stack = stack.Push(prop);
+            stack = properties.Aggregate(stack, (current, prop) => current.Push(prop));
```
Comment (nblumhardt) [inline src/Serilog/Context/LogContext.cs]: Not sure this is idiomatic use of `Aggregate` here  - should be equivalent but maybe the original was clearer?

File: src/Serilog/Settings/AppSettings/AppSettingsSettings.cs [hunk 2]
```diff
@@ -29,7 +29,6 @@ class AppSettingsSettings : ILoggerSettings
         public AppSettingsSettings(string settingPrefix = null)
         {
             _settingPrefix = settingPrefix == null ? "serilog:" : $"{settingPrefix}:serilog:";
-            System.Console.WriteLine(_settingPrefix);
```
Comment (nblumhardt) [inline src/Serilog/Settings/AppSettings/AppSettingsSettings.cs]: Eek! Nice catch.

File: src/Serilog/Context/LogContext.cs (hunk 1 above)
Comment (khellang) [inline src/Serilog/Context/LogContext.cs]: Huh, I guess R# suggest this and I just Alt+Enter'd my way through :stuck_out_tongue_closed_eyes: I'll revert...

**Comment by nblumhardt** [comment]
Looks great! Definitely think this will avoid an endless stream of small updates in other changes.

Just one comment in there, otherwise :shipit: :-)

**Comment by khellang** [comment]
Alright. Rebased and removed :smile:

**Comment by nblumhardt** [comment]
:+1:

**Pull request merged**